Packs/ArmisEventCollector/Integrations/ArmisEventCollector/ArmisEventCollector.py

```diff
@@ -0,0 +1,427 @@
+import demistomock as demisto
+from CommonServerPython import *
+import urllib3
+from typing import Any
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%S'
+VENDOR = 'armis'
+PRODUCT = 'security'
+API_V1_ENDPOINT = '/api/v1'
+DEFAULT_MAX_FETCH = 1000
+EVENT_TYPES = {
+    'Alerts': {
+        'unique_id_key': 'alertId',
+        'aql_query': 'in:alerts',
+        'type': 'alerts'},
+    'Threat activities': {
+        'unique_id_key': 'activityUUID',
+        'aql_query': 'in:activity type:"Threat Detected"',
+        'type': 'threat_activities'}}
```

Comment:
You can consider using NamedTuple here and it will allow you a nicer access to the event's properties. (event.type or event.aql_query)

Reply:
Didn't thought of that, using NamedTuple is much cleaner!